coding_style_and_standards.md

```diff
@@ -33,6 +34,19 @@ maintained by the ManageIQ team.
   in the [.rubocop_base.yml](.rubocop_base.yml) file, which is inherited by most
   projects in the ManageIQ organization.
 
+## Documentation
+
+We chose [yardoc](https://yardoc.org/) to create inline code documentation. For now documentation is scarce but
+we hope the situation will improve. This is why we limit the created documentation to those files that are well
+documented. Otherwise it would be hard to find valuable documentation in a lot of autogenerated method signatures.
+Please try to document methods that are used by external teams, like providers.
+These are the integration API for them and should be documented best.
+
+To view the documentation online visit [rubydoc.info](http://www.rubydoc.info/github/manageiq/manageiq). For a local
```

Comment:
Good enough until we make a proper API documentation portal (I'm all for an `api.manageiq.org` eventually)